client/packages/common/src/ui/components/navigation/AppNavLink/AppNavLink.tsx

```diff
@@ -72,6 +76,9 @@ export interface AppNavLinkProps {
   to: string;
   visible?: boolean;
   onClick?: () => void;
+  requireConfirmation?: boolean;
+  confirmMessage?: string;
+  confirmTitle?: string;
 }
```

Comment:
Hey Carl,

I'm getting a situation where the onClick method won't be handled in AppNavLink when it is passed inactive.

And it also expects a drawer - so the UI is a little strange.

![10D5BABE-89F7-4154-ACC9-4D5C828A37C3](https://github.com/msupply-foundation/open-msupply/assets/90807420/3f20ea40-073c-4f93-853f-fab9e41e0f90)

Where you thinking of doing the above but also adding conditional rendering in AppNav?

Reply:
Sorry for the late reply.

Yeah, I see what you mean. I guess it's strictly speaking not `inactive` since we want it to respond to a click event. What you can do is, rather than making it `inactive`, just set the `to` prop to "#", which makes it do nothing as a regular link, but will still run the onClick method:
```tsx
<AppNavLink
  to={'#'}
  icon={<PowerIcon fontSize="small" color="primary" />}
  text={t('logout')}
  onClick={() => showConfirmation({})}
/>
```

Then you don't need the `<Box>` component either :)